src/Components/CreateImageWizardV2/steps/FileSystem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Text, Form, Title } from '@patternfly/react-core';
+
+import FileSystemAutomaticPartition from './FileSystemAutomaticPartitionInformation';
+import FileSystemConfiguration from './FileSystemConfiguration';
+import FileSystemPartition from './FileSystemPartition';
+
+import { useAppSelector } from '../../../../store/hooks';
+import { selectFileSystemPartitionMode } from '../../../../store/wizardSlice';
+export type FileSystemPartitionMode = 'automatic' | 'manual' | 'oscap';
```

Comment:
I'm still not very sure about adding 'oscap' as a partitioning mode, alas can't think of better solution right now :thinking: 

What we'll need to keep in mind is that the oscap would need to be parsed as "manual" in requestMapper.